.github/workflows/pythonapp.yml

```diff
@@ -27,4 +27,5 @@ jobs:
     - name: Test with pytest
       run: |
         pip install pytest
-        pytest
+        # Set pytest to ignore any DeprecationWarnings, as WTForms is generating 26 warnings
```

Comment:
Run `make dev` or `make prod` before running tests.

Reply:
Add a new step called something along the lines of setup project.

This step runs 
```
make dev
python3 manage.py initdb
```

Both of these are required before the application can be run.